roles/nuage_node/templates/vsp-openshift.j2

```diff
@@ -9,7 +9,7 @@ enterpriseName: {{ enterprise }}
 # Name of the domain in which pods will reside
 domainName: {{ domain }}
 # Name of the VSD user in admin group
-vsdUser: {{ vsduser }}
+vsdUser: {{ vsd_user }}
```

Comment:
vsd_user | default(vsd_user) to preserve backwards compatibility? Your call not sure how much backwards compat matters to you.